lib/modules/multiredditNavbar.js

```diff
@@ -0,0 +1,117 @@
+addModule('multiredditNavbar', (module, moduleID) => {
+	module.moduleName = 'Multireddit Navigation';
+	module.description = 'Enhance the navigation bar shown on the left side of the frontpage.';
+	module.category = 'Subreddits';
+
+	module.includes = [
+		RESUtils.regexes.frontpage
+	];
+
+	Object.assign(module.options, {
+		sectionMenu: {
+			type: 'boolean',
+			value: true,
+			description: 'Show a menu linking to various sections of the multireddit when hovering your mouse over the link.'
+		},
+		sectionLinks: {
+			dependsOn: 'sectionMenu',
+			type: 'table',
+			addRowText: '+add multireddit section shortcut',
+			fields: [{
+				name: 'label',
+				type: 'text'
+			}, {
+				name: 'url',
+				type: 'text'
+			}],
+			value: [
+				['new', './new'],
+				['rising', './rising'],
+				['controversial', './controversial'],
+				['top', './top'],
+				['top this month', './top?t=month'],
+				['gilded', './gilded'],
+				['promoted', './ads'],
+			]
+		},
+		hoverDelay: {
+			dependsOn: 'sectionMenu',
+			type: 'text',
+			value: 1000,
+			description: 'Delay, in milliseconds, before hover tooltip loads. Default is 1000.',
+			advanced: true
+		},
+		fadeDelay: {
+			dependsOn: 'sectionMenu',
+			type: 'text',
+			value: 200,
+			description: 'Delay, in milliseconds, before hover tooltip fades away. Default is 200.',
+			advanced: true
+		},
+		fadeSpeed: {
+			dependsOn: 'sectionMenu',
+			type: 'text',
+			value: 0.7,
+			description: 'Fade animation\'s speed (in seconds). Default is 0.7.',
+			advanced: true
+		}
+	});
+
+	module.go = function() {
+		const $multis = $('.listing-chooser .multis');
+
+		if (module.options.sectionMenu.value) {
+			$multis.on('mouseover', 'li', onMouseoverMultiLink);
+		}
+	};
+
+	function onMouseoverMultiLink(e) {
+		const link = e.currentTarget.querySelector('a[href^="/me/m"]');
+		if (!link) {
+			return;
+		}
+		modules['hover'].dropdownList(moduleID)
+			.target(e.currentTarget)
+			.options({
+				openDelay: module.options.hoverDelay.value,
+				fadeDelay: module.options.fadeDelay.value,
+				fadeSpeed: module.options.fadeSpeed.value,
+				pin: modules['hover'].pin.right,
+				offsetWidth: -10,
+				offsetHeight: 1,
+				bottomPadding: 0,
+			})
+			.populateWith(() => populateSectionMenu(link.href))
+			.begin();
+	}
+
+	const populateSectionMenu = (baseUrl) => [
+		module.options.sectionLinks.value
+			.map((...args) => populateSectionItem(baseUrl, ...args))
+			.reduce((prev, curr) => (curr ? prev.add(curr) : prev), $())
+			.add(populateSectionItem('.', [
+				`<i>${module.moduleName}</i>`,
+				modules['settingsNavigation'].makeUrlHash(moduleID, 'sectionMenu')
+			]))
+	];
+
+	function populateSectionItem(baseUrl, link) {
+		if (!(link && link.length >= 2)) {
+			return;
```

Comment:
Errors from the `consistent-return` lint can be a bit misleading; it wants all or none of the return statements to return a value (`undefined` is allowed, as long as it's explicit).